### ISSUE TITLE
Scrolling profile timeline causes overlapping

### ISSUE THREAD

**Issue by Danysan1** [mastodon/mastodon-android#619]
Sometimes clicking on my icon I get correctly taken to my timeline but when scrolling posts begin overlapping:

https://github.com/mastodon/mastodon-android/assets/8406735/037680b7-87e1-4144-af86-30355d9edfec

This is happening on Android 13 on Pixel  6a. I haven't found a reliable path to reproduce this.
When this happens, going back to the app home and going again to my timeline is not enough: the problem persists.
The problem solves only by closing entirely the app and restarting it.

**Comment by grishka** [comment]
Uh what. This should be impossible.

**Comment by sk22** [comment]
i'm able to reproduce this by enabling "don't keep activities" in the developer settings.

https://github.com/mastodon/mastodon-android/assets/6217438/5bd7ed7c-6cbb-4354-bf60-1aecc544772a

**Comment by sk22** [comment]
so i'm assuming, if the saved activity gets restored, a second view is also placed above it?

**Comment by sphars** [comment]
I was able to replicate this issue. While not super rigorous, I managed to trigger it twice. 

- Go to a profile's timeline
- Scroll down for a bit
- Leave the app (still leaving it in Recents)
- Go back to the app via Recents after some time (an hour for me)
- Issue appears

So yeah, it appears to do with the activity lifecycle.

Google Pixel 4a, Android 13

https://github.com/mastodon/mastodon-android/assets/18315081/0123daa0-b678-404c-bd37-414792ec4c80
